lib/common/digest.c

```diff
@@ -226,37 +226,22 @@ pcmk__xa_filterable(const char *name)
 char *
 crm_md5sum(const char *buffer)
 {
-    unsigned int dlen = gnutls_hash_get_len(GNUTLS_DIG_MD5);
-    unsigned char *raw_digest = NULL;
     char *digest = NULL;
-    int rc = 0;
-
-    if (dlen == 0) {
-        return NULL;
-    }
+    gchar *raw_digest = NULL;
 
     if (buffer == NULL) {
         return NULL;
     }
 
-    raw_digest = pcmk__assert_alloc(dlen, sizeof(unsigned char));
-
-    rc = gnutls_hash_fast(GNUTLS_DIG_MD5, buffer, strlen(buffer), raw_digest);
+    raw_digest = g_compute_checksum_for_string(G_CHECKSUM_MD5, buffer, strlen(buffer));
```

Comment:
We could just use -1 for the length since it's nul-terminated

Reply:
Fixed.